lib/flood_risk_engine/services/address/deserialize/deserialize_base.rb

```diff
@@ -0,0 +1,19 @@
+# Base class for Deserializers - transform INBOUND data into
+# data suitable to construct an Address instance
+#
+module FloodRiskEngine
+  module Services
+    module Address
+      module Deserialize
+
+        class DeserializeBase
+
+          def address_data(_inbound)
+            raise NotImplementedError, "No address_data implementation provided"
+          end
+
+        end
+      end
+    end
+  end
+end
```

Comment:
I could never see the point of this class in WEX ;-) (there's only one ever one sublcass) - I'd delete it if it were me